include/enums.h

```diff
@@ -3378,4 +3384,42 @@ enum RushFlags {
     RUSH_FLAG_POWER = 2,
 };
 
+enum FileMenuMessages {
+    FILE_MESSAGE_NONE                       = 0,
+	FILE_MESSAGE_SELECT_FILE_TO_START       = 1, // Select file to start:[End]
```

Comment:
these should be 4 space-indented but every line except the first uses tab characters